react/week3/todo-app/src/Components/Border.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+function Border(props) {
+  return(
+    <div style={{ border: "2px solid #000", margin: "1em" }}>
```

Comment:
please avoid using inline styles, use className or id instead  and style in the CSS files, or if you want please check this https://styled-components.com/
it is one of the best style library in ReactJs and will be a plus in your cv